hedera-node/hedera-app/src/xtest/java/contract/AbstractContractXTest.java

```diff
@@ -115,6 +142,35 @@ protected void handleAndCommit(@NonNull final TransactionHandler handler, @NonNu
         }
     }
 
+    protected HandleContext handleAndCommitSingleTransaction(
```

Comment:
The idea was that I would like to return the handlecontext back to the caller so that I can assert something on it.  The problem is that the prng call does not change state but does add a record builder to the list which has the salt value I am looking for.  I asked at the modularization call whether the values in the record builders are available and they are looking into the possibility.